versions/management/commands/import_reviews.py

```diff
@@ -0,0 +1,241 @@
+                            "is_most_recent": True,
+                        }
+                    )
+
+            # If no results were found, use the text content of the cell
+            if not results_data and (description := result_cell.text.strip()):
+                results_data.append({"short_description": description})
+
+        reviews.append((review_data, results_data))
+
+    return reviews
+
+
+def _parse_users_from_raw_names(raw_name_string: str) -> list[tuple[str, str]]:
+    """
+    Parse a raw name string into a list of (first_name, last_name) tuples.
+    Handles inputs like:
+        "John Doe"
+        "John Doe & Jane Smith"
+        "John Doe and Jane Smith"
+        "John Doe, Jane Smith"
+        "John Doe, Jane Smith, Joaquin M López Muñoz"
+
+    Returns a list like:
+        [("John", "Doe"), ("Jane", "Smith"), ("Joaquin M López", "Muñoz")]
+    """
+    # Clean up the string - normalize whitespace and separators
+    cleaned = (
+        raw_name_string.replace("\n", " & ")
+        .replace("\t", " ")
+        .replace(" and ", " & ")
+        .replace(",", " & ")
+        .replace("ª", "")  # special character observed
+        .replace("OvermindDL1", "")  # replaced review manager observed
+    )
+
+    # Collapse multiple `&` separators
+    while " & & " in cleaned:
+        cleaned = cleaned.replace(" & & ", " & ")
+
+    # Collapse multiple spaces
+    cleaned = " ".join(cleaned.split())
+
+    # Split on & and clean up each name
+    names = [name.strip() for name in cleaned.split("&")]
+    parsed_names = []
+
+    for name in names:
+        if not name:
+            continue
+
+        # Try to split into first and last name
+        parts = name.split()
+        if len(parts) == 1:
+            # Just one name, treat as first name
+            parsed_names.append((parts[0], ""))
+        else:
+            # Assume last word is last name, rest is first name
+            parsed_names.append((" ".join(parts[:-1]), parts[-1]))
+
+    return parsed_names
+
+
+def _get_user_from_name(first_name, last_name):
+    matching_users = User.objects.filter(
+        Q(first_name__iexact=first_name, last_name__iexact=last_name)
+        | Q(
+            first_name__unaccent__iexact=first_name,
+            last_name__unaccent__iexact=last_name,
+        )
+        | Q(display_name__unaccent__iexact=f"{first_name} {last_name}")
+        | Q(display_name__iexact=f"{first_name} {last_name}")
+    )
+    if count := matching_users.count() == 1:
+        return matching_users.first()
+    elif count:
+        click.secho(
+            f"Found multiple users with the same name: {first_name} {last_name}",
+            fg="red",
+        )
+        return None
+
+    # No existing user by this name; create a fake "stub" user
+    fake_email = generate_fake_email(f"{first_name} {last_name}")
+    if user := User.objects.filter(email=fake_email).first():
+        return user
+    return User.objects.create_stub_user(
```

Comment:
This is an area I'm unfamiliar with, is there a process for a real user to claim a stub user's data?

Reply:
Apparently so. I haven't tested it, but there are multiple references in the documentation/comments about "claiming" stub accounts.